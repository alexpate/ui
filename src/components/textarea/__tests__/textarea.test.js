/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';

import Textarea from '../';

describe('Input', () => {
  const defaultProps = {
    size: 'medium',
    theme: 'default',
  };

  test('should render shallow component ok', () => {
    const element = renderer.create(<Textarea {...defaultProps} />).toJSON();
    expect(element).toMatchSnapshot();
  });
});
