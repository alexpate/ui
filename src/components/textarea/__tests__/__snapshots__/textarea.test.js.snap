// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render shallow component ok 1`] = `
<textarea
  className="css-170r1en css-gspz1f0"
  disabled={undefined}
  onChange={undefined}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onScroll={[Function]}
  readOnly={undefined}
  size="medium"
  style={
    Object {
      "height": undefined,
    }
  }
/>
`;
